MAUI/Picker/getting-started.md

```diff
@@ -0,0 +1,308 @@
+
+using Syncfusion.Maui.Picker;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfPicker picker = new SfPicker();
+        this.Content = picker;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Register the handler
+
+The `Syncfusion.Maui.Core` NuGet is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
+
+{% tabs %}
+{% highlight C# tabtitle="MauiProgram.cs" hl_lines="1 10" %}
+
+using Syncfusion.Maui.Core.Hosting;
+namespace GettingStarted
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+
+            builder.ConfigureSyncfusionCore();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+                fonts.AddFont("Segoe-mdl2.ttf", "SegoeMDL2");
+            });
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set header to the Picker
+
+The SfPicker control allows you to add the header text by setting the [Text] property in the [PickerHeaderView]. To enable the header view by setting the [Height] property in [PickerHeaderView].
+
+{% tabs %}
+{% highlight xaml tabtitle="XAML" hl_lines="3" %}
+
+<picker:SfPicker x:Name="picker">
+    <picker:SfPicker.HeaderView>
+        <picker:PickerHeaderView Text="Select a color" Height="40" />
+    </picker:SfPicker.HeaderView>
+</picker:SfPicker>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="4 5" %}
+
+SfPicker picker = new SfPicker();
+picker.HeaderView = new PickerHeaderView()
+{
+    Text = "Select a color",
+    Height = 40,
+};
+
+this.Content = picker;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Add picker items to the picker
+
+The .NET MAUI Picker control is a data bounded control. Hence, you must create collection of data’s and bind it to the picker control.
+
+* Create a simple Observable Collection of data for the picker,
+
+{% tabs %}
+{% highlight c# tabtitle="ItemInfo.cs" %}
+
+public class ItemInfo
+{
+    private ObservableCollection<object> dataSource = new ObservableCollection<object>()
+    {
+        "Pink", "Green", "Blue", "Yellow", "Orange", "Purple", "SkyBlue", "PaleGreen"
+    };
+
+    public ObservableCollection<object> DataSource
+    {
+        get
+        {
+            return dataSource;
+        }
+        set
+        {
+            dataSource = value;
+        }
+    }
+
+    public ItemInfo()
+    {
+
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+* Bind the Collection to picker
+
+Picker control allows you to bind the collection of data by setting the [ItemsSource] property from [PickerColumn]. You can bind the collection of data in both XAML or C#.
```

Comment:
The Picker control allows you to bind data collection by setting the [ItemsSource] property from the [PickerColumn]. Bind data collection in both XAML and C#.

Reply:
Content modified.